examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/mix-precision/README.md

```diff
@@ -0,0 +1,65 @@
+# Run
+ 
+In this examples, you can verify the accuracy on HPU/CUDA device with emulation of MXFP4, MXFP8, NVFP4 and uNVFP4.
+
+## Requirement
+
+```bash
+# neural-compressor-pt
+INC_PT_ONLY=1 pip install git+https://github.com/intel/neural-compressor.git@xinhe/mx_recipe
+# auto-round
+pip install git+https://github.com/intel/auto-round.git@xinhe/llama_tmp
```

Comment:
same, why using autoround personal branch?  suppose user can reproduce with release version. if there is something only in this personal branch, it should be merged into autoround master/release branch

Reply:
removed